src/langcheck/metrics/metric_inputs.py

```diff
@@ -378,6 +378,8 @@ def get_standard_metric_inputs(
     | tuple[IndividualInputType, IndividualInputType] = None,
     reference_outputs: IndividualInputType
     | tuple[IndividualInputType, IndividualInputType] = None,
+    additional_params: dict[str, IndividualInputType] | None = None,
+    additional_params_to_prompt_var_mapping: dict[str, str] | None = None,
```

Comment:
Should we rename the "get_standard_metric_inputs" function to just "get_metric_inputs"? Since it's no longer just limited to the "standard" inputs